testfixtures/logcapture.py

```diff
@@ -106,17 +107,17 @@ def uninstall_all(cls):
         "This will uninstall all existing :class:`LogHandler` objects."
         for i in tuple(cls.instances):
             i.uninstall()
-        
+
```

Comment:
You know, I'm not sure I really care about this trailing whitespace.
If your editor highlights it, change your editor config.